src/vivarium/framework/state_machine.py

```diff
@@ -19,12 +19,12 @@
 if TYPE_CHECKING:
     from vivarium.framework.engine import Builder
     from vivarium.framework.population import PopulationView
-    from vivarium.types import Time
+    from vivarium.types import SimTime
 
 
 def _next_state(
     index: pd.Index,
-    event_time: "Time",
+    event_time: "SimTime",
```

Comment:
Why do we sometimes use quotes when defining types?

Reply:
Quotes are usually used when dealing with circular dependencies